gui/src/components/Taipy/NavBar.tsx

```diff
@@ -56,7 +56,7 @@ const NavBar = (props: LovProps) => {
     const linkChange = useCallback(
         (evt: React.SyntheticEvent, val: string) => {
             if (Object.keys(state.locations || {}).some((route) => val === route)) {
-                navigate(val);
+                navigate(val.substring(1));
```

Comment:
Is the target url always absolute?

Reply:
it should be absolute so the baseurl can take proper affect. i might be wrong but we can always fix this in the future :D